scripts.js

```diff
@@ -0,0 +1,17 @@
+var text = "Papugi koszą trawę, unosząc się nad ziemią 15 centymetrów, na czerwonych dywanach.";
+console.log(text);
+
+var animal = "Zielone słonie";
+console.log(animal);
+
+var animalUpperCased = animal.toUpperCase();
+console.log(animalUpperCased);
+
+var textReplace = text.replace("Papugi", animalUpperCased);
```

Comment:
Zrób tak żeby zamiast Papug, zamieniany był zawsze pierwszy wyraz zmiennej `text` - bez względu na to co tam będzie. Możesz użyć do tego metody `split` (google -> "mdn split")